src/bin/pg_autoctl/primary_standby.c

```diff
@@ -53,6 +53,7 @@ static void local_postgres_update_pg_failures_tracking(LocalPostgresServer *post
 	{ "log_connections", "off" }, \
 	{ "log_disconnections", "off" }, \
 	{ "log_lock_waits", "on" }, \
+	{ "password_encryption", NULL }, /* will be set later */ \
```

Comment:
```suggestion
	{ "password_encryption", NULL }, /* will be set later based on --auth */ \
```

Reply:
Opted to remove the comment as I set the default as `md5` now